### ISSUE TITLE
[Feature Request] New Command Open Graph Analysis in Default Type (toggle)

### ISSUE THREAD

**Issue by FelipeRearden** [SkepticMystic/graph-analysis#30]
Hello again @SkepticMystic  !!!!!

I have been watching you guys about this plugin and I finally get a time to start using and WOW !!!!! This is amazing !!!!!! What a great ideia !!!!!!! 

I’m just starting to understand all algorithms and reading all posts but I already know that this is a great plugin for me :)

I have two “tiny” FR + A discussion about a workflow that I will post to share with you. 

This first FR is simple to explain: A new command in the Command Palette to Open/Close (toggle) the Graph Analysis Panel in Default Type using a hotkey  !

It’s always a pleasure talking with you @SkepticMystic !!!! 

I wish you a fantastic day!

**Comment by SkepticMystic** [comment]
@FelipeRearden Hey 😄 
Thank you for the few new posts!

For this one, the `Open Graph Analysis View` already opens in the default type, right?
So you are looking for a command that would close it, too?

**Comment by FelipeRearden** [comment]
Hello @SkepticMystic !!!

I just updated for version `0.10.0` let me just gibe some feed back before answering your questions :)

01- In Settings Pane is not possible to choose a Default Type 

- I open the Menu List but nothing id there :) Maybe I did something wrong :)

Img

> For this one, the Open Graph Analysis View already opens in the default type, right?

- I think that why I'm not seeing `Open Graph Analysis View ` in my command palette 

Img 

02- When hitting the command `Graph Analysis: Open type_name` when the SIDE PANEL is OPEN and focused in another plugin like Backlinks (for example) does not open/show the Graph Analysis Panel
- The Command is working (the graph panel is changing to the type defined by the command, but the Side Panel is not showing) 

---

Sorry for my bad explanation in the FR. Let me rephrase :)

I'm looking for a command that have three features:

1- If the Side Panel is Closed it will Open/Show the Side Panel in the Graph Analysis Panel default type;

2- If the Side Panel is Open at another plugin like Backlinks or Outliner, after hitting the command the Side Panel will change to Graph Analysis in the default type;

3- Same behavior from `1` and `2` applied to the other commands: `Graph Analysis: Open type_name`.

> So you are looking for a command that would close it, too?

My First idea was to have a command that close the Graph panel but I think it's not necessary since get already have a command to close the left/right sidebars :)


Have a great day!

**Comment by SkepticMystic** [comment]
@FelipeRearden the settings bug should be fixed, discussed in #36.
Regarding the FR here, I can try do that!
So now that the setting bug is fixed, the `Open Analysis View` cmd should work (it only shows if the view isn't already open). If the view is already open, use the `Refresh Analysis View` cmd to make it active.

I can make it so that if another view is focused, running the cmd will make GA active

**Comment by FelipeRearden** [comment]
> @FelipeRearden the settings bug should be fixed, discussed in #36.

Great @SkepticMystic !!!! Working on `0.10.1` on Obsidian  `0.12.19`

> So now that the setting bug is fixed, the `Open Analysis View` cmd should work (it only shows if the view isn't already open). If the view is already open, use the `Refresh Analysis View` cmd to make it active.

Thanks for the info @SkepticMystic ! Cmd wasn't showing to me because  GA was already open :)

> Regarding the FR here, I can try do that!

😄 😄 😄 😄 😄 

> I can make it so that if another view is focused, running the cmd will make GA active

Thank you very much @SkepticMystic !!!! 

As a source of information the behavior that I was suggesting is similar to `Backlinks: Show Backlinks Pane` about showing the GA pane with the sidebar closed and open :)

Thanks again !!!!
